File: BrickKit/app/src/main/java/com/wayfair/brickkitdemo/MainActivity.java
/**
 * Copyright © 2017 Wayfair. All rights reserved.
 */
package com.wayfair.brickkitdemo;

import android.os.Bundle;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

/**
 * Main activity for the app.
 */
public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        setSupportActionBar(toolbar);

        if (savedInstanceState == null) {
            getSupportFragmentManager().beginTransaction().replace(R.id.content, new MainActivityFragment()).commit();
        }
    }
}

File: BrickKit/app/src/main/java/com/wayfair/brickkitdemo/MainActivity.kt
/**
 * Copyright © 2017-2021 Wayfair. All rights reserved.
 */
package com.wayfair.brickkitdemo

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.wayfair.brickkitdemo.mainfragment.MainActivityFragment

/**
 * Main activity for the app.
 */
class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        setSupportActionBar(findViewById(R.id.toolbar))

        if (savedInstanceState == null) {
            supportFragmentManager.beginTransaction().replace(R.id.content, MainActivityFragment()).commit()
        }
    }
}
